packages/app/test/src/features/home/application/feed_service_test.dart

```diff
@@ -25,10 +26,14 @@ void main() {
 
       // Act
       final postsToAdd = List.generate(
-        pageSize,
-        (index) => const PostEntity(
+        pageSize, //@lishaduck implemented a fix here but you may want to review
```

Comment:
> What fix?
> 
> 
> 
> Also, please make a draft PR. 

 _Originally posted by @lishaduck in [c090f45](https://github.com/PHS-TSA/nexus/commit/c090f457a414d4df05404c2071fb26e684dcc5f2#r147682001)_